executor/runtime/types/sidecars.go

```diff
@@ -33,6 +34,15 @@ var systemServices = []ServiceOpts{
 		UnitName:     "titus-sidecar-seccomp-agent",
 		Required:     true,
 		EnabledCheck: shouldStartTitusSeccompAgent,
+		Volumes: map[string]struct{}{
+			"/titus/systemdns": {},
+		},
+	},
+	{
+		ServiceName:  SidecarServiceSystemDNS,
+		UnitName:     "titus-sidecar-systemdns",
+		Required:     false,
```

Comment:
This needs to be required :(

Reply:
Noted. What does `Required` mean in this context actually?

Reply:
Check out the docstring for the struct ```/ If true, the startup of a task will fail, otherwise will just log```

Reply:
Ah got it -- so if the service is enabled but startup fails, the whole task will fail to start, otherwise the task will continue on without this service, IIUC?

Reply:
correct.

DNS is critical though. I don't want a task to start without it.
I would rather we fail fast and report the error back to the user (and have the failure show up on sidecar dashboards, etc)